specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json

```diff
@@ -0,0 +1,832 @@
+                "name": {
+                    "type": "string",
+                    "description": "The name of the azure function receiver. Names must be unique across all receivers within an action group."
+                },
+                "functionAppResourceId": {
+                    "type": "string",
+                    "description": "The azure resource id of the function app."
+                },
+                "functionName": {
+                    "type": "string",
+                    "description": "The function name in the function app."
+                },
+                "httpTriggerUrl": {
+                    "type": "string",
+                    "description": "The http trigger url where http request sent to."
+                }
+            },
+            "required": [
+                "name",
+                "functionAppResourceId",
+                "functionName",
+                "httpTriggerUrl"
+            ]
+        },
+        "ArmRoleReceiver": {
+            "description": "An arm role receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the arm role receiver. Names must be unique across all receivers within an action group."
+                },
+                "roleId": {
+                    "type": "string",
+                    "description": "The arm role id."
+                }
+            },
+            "required": [
+                "name",
+                "roleId"
+            ]
+        },
+        "ReceiverStatus": {
+            "type": "string",
+            "enum": [
+                "NotSpecified",
+                "Enabled",
+                "Disabled"
+            ],
+            "x-ms-enum": {
+                "name": "ReceiverStatus",
+                "modelAsString": false
+            },
+            "description": "Indicates the status of the receiver. Receivers that are not Enabled will not receive any communications."
+        },
+        "EnableRequest": {
+            "description": "Describes a receiver that should be resubscribed.",
+            "properties": {
+                "receiverName": {
+                    "type": "string",
+                    "description": "The name of the receiver to resubscribe."
+                }
+            },
+            "required": [
+                "receiverName"
+            ]
+        },
+        "ErrorResponse": {
+            "description": "Describes the format of Error response.",
+            "type": "object",
+            "properties": {
+                "code": {
+                    "description": "Error code",
+                    "type": "string"
+                },
+                "message": {
+                    "description": "Error message indicating why the operation failed.",
+                    "type": "string"
+                }
+            }
+        },
+        "ActionGroupPatchBody": {
```

Comment:
It would make it easier to update the receivers if Patch supported that. Else, everytime user needs to update some receiver - either add new receivers, or remove a receiver, or update some setting of already added receiver, they will have to re-PUT the entire action group again.